app/build.gradle

```diff
@@ -276,11 +276,11 @@ dependencies {
 
     // use specific version of data binding until google resolves the bugs with mixed case package names
     //noinspection GradleDependency
-    implementation 'com.android.databinding:adapters:3.1.4'
+    //implementation 'com.android.databinding:adapters:3.1.4'
```

Comment:
Please remove unused code. If necessary, commits can be reverted or partially restored from git history of a file.